src/bb_excel/core.clj

```diff
@@ -169,13 +172,14 @@
 (defn get-unique-strings
   "Get dictionary of all unique strings in the Excel spreadsheet"
   [^ZipFile zipfile]
-  (let [wb (.getEntry zipfile (str "xl/sharedStrings.xml"))
-        ins (.getInputStream zipfile wb)
-        x (parse-str (slurp ins))]
-    (->>
-     (filter #((:text-part tags) (:tag %)) (xml-seq x))
-     (map get-cell-text)
-     (zipmap (range)))))
+  (if-let [wb (.getEntry zipfile (str "xl/sharedStrings.xml"))]
```

Comment:
This is a realistically possible situation. I created such a file using LibreOffice and attached example file to the tests. I didn't do anything special to get it. And I also don't think this is a mistake. Google Docs opens this file normally.